apps/sms/js/threads.js

```diff
@@ -18,6 +18,10 @@
     this.messages = new Map();
   }
 
+  Thread.prototype.getDraft = function() {
```

Comment:
Also I think we can have ```hasDrafts``` as well later, when we have separate map(draftId<->draft) for threadless drafts so that hasDrafts can be really quick.